libbraiding.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: libbraiding
```

Comment:
I think you can use `@PACKAGE_NAME@` here

Reply:
fixed